### PR TITLE
Generate components for translate, transcribe, and my feeds.

### LINKED ISSUE #19: Generate components for translate and transcribe

### PR DESCRIPTION
**Pull request by jagjennings**
Create Translate, Transcribe with form fields and generate My Feeds Component.
- transcribe form: https://screenshot.googleplex.com/xJiH3p7CzuF.png
- translate form: https://screenshot.googleplex.com/fK2DZspiO6G.png

### CODE REVIEW

**Review by ERetana2 — DISMISSED** [review]

File: launchpod/angular-launchpod/src/app/create-form/create-form.component.ts [hunk 1]
```diff
@@ -16,7 +15,8 @@ export class CreateFormComponent implements OnInit {
 
   constructor(private formHandlerService: FormHandlerService) {}
 
-  ngOnInit(): void {}
+  ngOnInit(): void {
+  }
```
Comment (bramanudom) [inline launchpod/angular-launchpod/src/app/create-form/create-form.component.ts]: Revert back to one line?

File: launchpod/angular-launchpod/src/app/my-feeds-page/my-feeds-page.component.ts [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+  selector: 'app-my-feeds-page',
+  templateUrl: './my-feeds-page.component.html',
+  styleUrls: ['./my-feeds-page.component.css']
+})
+export class MyFeedsPageComponent implements OnInit {
+
+  constructor() { }
+
+  ngOnInit(): void {
+  }
```
Comment (bramanudom) [inline launchpod/angular-launchpod/src/app/my-feeds-page/my-feeds-page.component.ts]: Same here --  one line?

File: launchpod/angular-launchpod/src/app/my-feeds-page/my-feeds-page.component.ts [hunk 3]
```diff
@@ -0,0 +1,15 @@
+import { Component, OnInit } from '@angular/core';
+
+@Component({
+  selector: 'app-my-feeds-page',
+  templateUrl: './my-feeds-page.component.html',
+  styleUrls: ['./my-feeds-page.component.css']
+})
+export class MyFeedsPageComponent implements OnInit {
+
+  constructor() { }
```
Comment (bramanudom) [inline launchpod/angular-launchpod/src/app/my-feeds-page/my-feeds-page.component.ts]: nit: extra space between brackets

File: launchpod/angular-launchpod/src/app/transcribe-form/transcribe-form.component.ts [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+  styleUrls: ['./transcribe-form.component.css']
+})
+export class TranscribeFormComponent implements OnInit {
+
+  feedValue: string;
+
+  constructor(private formHandlerService: FormHandlerService) { }
+
+  ngOnInit(): void {
+  }
```
Comment (bramanudom) [inline launchpod/angular-launchpod/src/app/transcribe-form/transcribe-form.component.ts]: Same here with the nits above. If I'm missing something please let me know.

File: launchpod/angular-launchpod/src/app/transcribe-form/transcribe-form.component.ts [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+
+  // Sends input data to backend when user clicks create button.
+  public postFormData() {
+    //TODO: implement this method
```
Comment (bramanudom) [inline launchpod/angular-launchpod/src/app/transcribe-form/transcribe-form.component.ts]: Could we attach a bug/issue to this?

File: launchpod/angular-launchpod/src/app/translate-form/translate-form.component.html [hunk 6]
```diff
@@ -0,0 +1,22 @@
...
+
+    <h2>Select the New Language</h2>
+    <mat-form-field appearance="fill">
+    <mat-label>Language</mat-label>
+    <mat-select>
+      <mat-option *ngFor="let language of languages" [value]="language.value">
+        {{language.viewValue}}
+      </mat-option>
+    </mat-select>
+  </mat-form-field>
```
Comment (bramanudom) [inline launchpod/angular-launchpod/src/app/translate-form/translate-form.component.html]: Please align indentation.

File: launchpod/angular-launchpod/src/app/translate-form/translate-form.component.ts [hunk 7]
```diff
@@ -0,0 +1,34 @@
...
+  ];
+
+  constructor() { }
+
+  ngOnInit(): void {
+  }
+
+  // Sends input data to backend when user clicks create button.
+  public postFormData() {
+    //TODO: implement this method
```
Comment (bramanudom) [inline launchpod/angular-launchpod/src/app/translate-form/translate-form.component.ts]: Please add issue here too.

**Review by bramanudom — CHANGES_REQUESTED** [review]

File: launchpod/angular-launchpod/src/app/create-form/create-form.component.ts (hunk 1 above)
Comment (jagjennings) [reply launchpod/angular-launchpod/src/app/create-form/create-form.component.ts]: sorry, generating the component from the cli makes it look like this lol. fixed it!

File: launchpod/angular-launchpod/src/app/my-feeds-page/my-feeds-page.component.ts (hunk 2 above)
Comment (jagjennings) [reply launchpod/angular-launchpod/src/app/my-feeds-page/my-feeds-page.component.ts]: done.

File: launchpod/angular-launchpod/src/app/my-feeds-page/my-feeds-page.component.ts (hunk 3 above)
Comment (jagjennings) [reply launchpod/angular-launchpod/src/app/my-feeds-page/my-feeds-page.component.ts]: done.

File: launchpod/angular-launchpod/src/app/transcribe-form/transcribe-form.component.ts (hunk 4 above)
Comment (jagjennings) [reply launchpod/angular-launchpod/src/app/transcribe-form/transcribe-form.component.ts]: done.

File: launchpod/angular-launchpod/src/app/transcribe-form/transcribe-form.component.ts (hunk 5 above)
Comment (jagjennings) [reply launchpod/angular-launchpod/src/app/transcribe-form/transcribe-form.component.ts]: done.

File: launchpod/angular-launchpod/src/app/translate-form/translate-form.component.ts (hunk 7 above)
Comment (jagjennings) [reply launchpod/angular-launchpod/src/app/translate-form/translate-form.component.ts]: done.

File: launchpod/angular-launchpod/src/app/translate-form/translate-form.component.html (hunk 6 above)
Comment (jagjennings) [reply launchpod/angular-launchpod/src/app/translate-form/translate-form.component.html]: done.

File: launchpod/angular-launchpod/src/app/app.component.html [hunk 8]
```diff
@@ -1,38 +1,37 @@
...
-          </a>
-      </mat-nav-list>
+  <mat-sidenav
+    #drawer
+    class="sidenav"
+    fixedInViewport
+    [attr.role]="(isHandset$ | async) ? 'dialog' : 'navigation'"
+    [mode]="'over'"
+    [opened]="false"
+  >
```
Comment (bramanudom) [inline launchpod/angular-launchpod/src/app/app.component.html]: Let's keep this on line 8.

File: launchpod/angular-launchpod/src/app/login-popup/login-popup.component.ts [hunk 9]
```diff
@@ -0,0 +1,15 @@
...
+  selector: 'app-login-popup',
+  templateUrl: './login-popup.component.html',
+  styleUrls: ['./login-popup.component.css']
+})
+export class LoginPopupComponent implements OnInit {
+
+  constructor() { }
+
+  ngOnInit(): void {
+  }
```
Comment (bramanudom) [inline launchpod/angular-launchpod/src/app/login-popup/login-popup.component.ts]: Nit: extra space and let's keep {} on the same line

**Review by bramanudom — DISMISSED** [review]

File: launchpod/angular-launchpod/src/app/app.component.html (hunk 8 above)
Comment (jagjennings) [reply launchpod/angular-launchpod/src/app/app.component.html]: done.

File: launchpod/angular-launchpod/src/app/login-popup/login-popup.component.ts (hunk 9 above)
Comment (jagjennings) [reply launchpod/angular-launchpod/src/app/login-popup/login-popup.component.ts]: done.

**Review by kassenq — APPROVED** [review]

**Review by ERetana2 — APPROVED** [review]

**Pull request merged**